ext/sodium/php_libsodium.h

```diff
@@ -34,8 +34,38 @@ extern zend_module_entry sodium_module_entry;
 
 #define SODIUM_CRYPTO_SIGN_KEYPAIRBYTES() crypto_sign_SECRETKEYBYTES + crypto_sign_PUBLICKEYBYTES
 
+#if SODIUM_LIBRARY_VERSION_MAJOR > 9 || (SODIUM_LIBRARY_VERSION_MAJOR == 9 && SODIUM_LIBRARY_VERSION_MINOR >= 6)
+
+/**
+ * MEMLIMIT is normalized to KB even though sodium uses Bytes in order to
+ * present a consistent user-facing API.
+ *
+ * Threads are fixed at 1 by libsodium.
+ *
+ * When updating these values, synchronize ext/standard/php_password.h values.
+ */
+#if defined(PHP_PASSWORD_ARGON2_MEMORY_COST)
+#define PHP_SODIUM_PWHASH_MEMLIMIT PHP_PASSWORD_ARGON2_MEMORY_COST
+#else
+#define PHP_SODIUM_PWHASH_MEMLIMIT (64 << 10)
+#endif
+#if defined(PHP_PASSWORD_ARGON2_TIME_COST)
+#define PHP_SODIUM_PWHASH_OPSLIMIT PHP_PASSWORD_ARGON2_TIME_COST
+#else
+#define PHP_SODIUM_PWHASH_OPSLIMIT 4
+#endif
+#if defined(PHP_SODIUM_PWHASH_THREADS)
+#define PHP_SODIUM_PWHASH_THREADS PHP_SODIUM_PWHASH_THREADS
+#else
+#define PHP_SODIUM_PWHASH_THREADS 1
+#endif
+
+#endif
+
 PHP_MINIT_FUNCTION(sodium);
+#if SODIUM_LIBRARY_VERSION_MAJOR > 9 || (SODIUM_LIBRARY_VERSION_MAJOR == 9 && SODIUM_LIBRARY_VERSION_MINOR >= 6)
```

Comment:
This change causes the compiler failure. If you get rid of it, tests should pass :)

Reply:
You were right, that was it. Thanks for your help.